src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala

```diff
@@ -27,7 +27,7 @@ import scala.util.Try
   */
 trait HeadersProcessor extends ToDownloadProcessor with ScorexLogging with ScorexEncoding {
 
-  private val charsetName = "UTF-8"
+  protected val charsetName = "UTF-8"
```

Comment:
It might be useful to move it to constants

Reply:
Moved it to `ErgoHistory` object as encoding is not used outside `history` package.